src/Symfony/Component/Form/ResolvedFormType.php

```diff
@@ -205,8 +205,16 @@ public function getOptionsResolver()
 
             $this->innerType->setDefaultOptions($this->optionsResolver);
 
+            if (method_exists($this->innerType, 'configureOptions')) {
```

Comment:
But this is on purpose to avoid a potential bc break. If someone is not using the abstract type and rather uses a custom implementation of the `FormTypeInterface` we might break applications. Therefore I decided to check if configureOptions() exists and if so calling configureOptions() additionally. I could not come up with a better way. Therefore I did not call configureOptions within the `AbstractType::setDefaultOptions()`.